bin/citgm-all.js

```diff
@@ -38,7 +38,12 @@ const yargs = commonArgs(require('yargs'))
   .option('excludeTags', {
     type: 'array',
     description: 'Define which tags from the lookup to skip'
-  });
+  })
+  .example('citgm-all -t /path/to/output.tap',
```

Comment:
should these unrelated examples come in a separate commit?

Reply:
Sure! I added them because otherwise I thought it looked a bit odd to only have examples of including tags, but I'll split them out.